test/apps/iodemo/io_demo.cc

```diff
@@ -764,11 +815,35 @@ class DemoClient : public P2pDemoCommon {
             if (opts().window_size == 1) {
                 std::cout << ", average latency: " << latency_usec << " usec";
             }
+            if (opts().print_conn_status) {
+                long min = _conn[0].num_completed;
+                long max = _conn[0].num_completed;
+                long avg = 0;
+                for (int i = 0; i < _conn.size(); i++) {
+                    if (_conn[i].num_completed < min) {
+                        min = _conn[i].num_completed;
+                    }
+                    if (_conn[i].num_completed > max) {
+                        max = _conn[i].num_completed;
+                    }
+                    avg += _conn[i].num_completed;
+                }
```

Comment:
can use std::accumulate

Reply:
@evgeny-leksikov Here we want to accumulate all ``_conn[i].num_completed``, but not ``_conn[i]``. ``_conn[i].num_completed`` is not a iterator. So maybe we can't use std::accumulate here.